### ISSUE TITLE
Cropping Layers

### ISSUE THREAD

**Issue by JadBatmobile** [sadeepj/crfasrnn_keras#59]
Can you explain the purpose of the Cropping2D layers?

**Comment by sadeepj** [comment]
Due to the padding operations done in the earlier layers of FCN network (see, for example, https://github.com/sadeepj/crfasrnn_keras/blob/master/src/crfrnn_model.py#L46), the output segmentation masks/logits have a boarder around them and are not aligned with the input image. Cropping2D layers are used to remove these boarders. Please refer to the FCN paper for more details.
